api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -65,9 +181,8 @@
         final Sort sort = new Sort(orderParam, columnParam);
```

Comment:
I think if you sort by status, a secondary sort ought to be on participant ID... otherwise it seems like the data will come back in a non-deterministic fashion?